share/picongpu/tests/openPMD-viewer-test/lib/python/test/param-test.py

```diff
@@ -0,0 +1,235 @@
+import argparse
+import sys
+import numpy as np
+import scipy.constants as spc
+from openpmd_viewer import OpenPMDTimeSeries
+import openpmd_viewer
+
+print(openpmd_viewer.__version__)
+parser = argparse.ArgumentParser(description="1")
+
+
+parser.add_argument(
+    "-r",
+    help="Path of the folder where the results"
+    " of the test-suite should be saved",
+    dest="data",
+    type=str,
+)
+
+args = parser.parse_args()
+
+# Initialize the OpenPMDTimeSeries object with the path to the data
+ts = OpenPMDTimeSeries(args.data)
+
```

Comment:
please define
```suggestion

base_density = 1.0e25
```
and use it in the dictionary below